code/style.css

```diff
@@ -1,7 +1,277 @@
+  padding: 5px;
+  font-size: 14px;
+}
+
+.input {
+  padding: 5px;
+  outline: none;
+}
+
+.input:focus {
+  border: 3px solid #353132;
+}
+
+.checkbox {
+  margin-top: 10px;
+}
+
+.button {
+  width: 30%;
+  padding: 5px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  background-color: #2b9348;
+  cursor: pointer; 
+  border: none;
+  text-decoration: none;
+  font-family: 'Raleway', sans-serif;
+  color: #f4f4ed;
+  font-size: 15px;
+  font-weight: bold;
+  text-transform: uppercase;
+  opacity: 1;
+  border: 0.5px solid #353132;
+  transition: 0.3s;
+}
+
+.button:hover {
+  background-image: linear-gradient(rgba(0, 0, 0, 0.4) 0 0);
+  opacity: 0.75;
+}
+
+/* Accordion */
+
+.accordion {
+  width: 95vw;
+  height: auto;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #f72585;
+  border-radius: 4px;
+  box-shadow: 0 0 5px rgba(0,0,0,0.3);
+  margin-top: 20px;
+  transition: 0.4s;
+}
+
+.question {
+  width: 100%;
+  height: 50px;
+  background-color: #2b9348;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+  position: relative;
+  border-top: 0.5px solid #353132;
+}
+
+.question-text {
+  font-size: 14px;
+  font-weight: bold;
+}
+
+.question::after {
+  content: "\002B";
+  position: absolute;
+  right: 0px;
+  margin-right: 20px;
+}
+
+.answer {
+  width: 100%;
+  height: 80px;
+  background-color: #f4f4ed;
+  display: none;
+}
+
+.answer-text {
+  text-align: center;
+  font-size: 14px;
+  color: #f72585;
+}
+
+.active, .question:hover {
+  background-image: linear-gradient(rgba(0, 0, 0, 0.4) 0 0);
+}
+
+.active::after {
+  content: "\2212";
+}
 
 .active + .answer {
-  /* some styling here */
+  display: block;
 }
+
+
+@media (min-width: 768px) {
+
+  .body {
+    display: flex;
+    justify-content: center;
+  }
+
+  .hero-image {
+    background-position: center;
+    justify-content: flex-start;
+    padding-left: 50px;
+    height: 60vh;
+  }
+
+  .title {
+    font-size: 64px;
+    background: none;
+    box-shadow: none;
+  }
+
+  .smaller-text {
+    font-size: 36px;
+    background: none;
+    box-shadow: none;
+  }
+
+  .signup-accordion-container {
+    flex-direction: row;
+    height: 70vh;
+    justify-content: space-evenly;
+    align-items: baseline;
+    margin-top: 0;
+
+  }
+
+  .signup-form {
+    width: 30vw;
+    height: auto;
+    margin-top: 30px;
+  }
+  
+  .h2-title {
+    font-size: 30px;
+  }
+
+  .label {
+    padding: 10px;
+    font-size: 18px;
+  }
+
+  .input {
+    padding: 10px;
+    background: #f4f4ed;
+  }
+  
+  .checkbox {
+    margin-top: 20px;
+  }
+  
+  .button {
+    padding: 10px;
+    margin-top: 20px;
+    margin-bottom: 20px;
+    width: 70%;
+  }
+
+  .accordion {
+    width: 30vw;
+    height: auto;
+    margin-top: 30px;
+  }
+
+  .question {
+    height: 100px;
+  }
+
+  .question-text {
+    font-size: 18px;
+  }
+
+  .answer {
+    height: 100px;
+  }
+  .answer-text {
+    font-size: 18px;
+  }
+
```

Comment:
I like how structured you are in your code and it is very easy to follow. Great job!!